src/Runner.Worker/ExecutionContext.cs

```diff
@@ -510,6 +511,23 @@ public void AddIssue(Issue issue, string logMessage = null)
                 }
 
                 _record.WarningCount++;
+            } else if (issue.Type == IssueType.Notice) 
+            {
+
+                // tracking line number for each issue in log file
```

Comment:
We should add validation on column/character length like dotcom does so we can surface a better error message to the user. EX: cannot provide column if start/end line are different.

Reply:
oh right. I need to add that.